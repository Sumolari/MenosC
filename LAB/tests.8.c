{
	int numero[ 7];
	numero = 20;
	if ( numero >= 4 ) numero = 3; else numero = 5 + 5;
}