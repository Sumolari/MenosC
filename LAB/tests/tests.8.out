Linea 3: Tipo incompatible
Linea 4: Tipo incompatible
Linea 4: Tipo incompatible
Linea 4: Tipo incompatible

Numero de errores:      4
